tests/link/numba/test_elemwise.py

```diff
@@ -119,6 +121,11 @@ def test_Elemwise(inputs, input_vals, output_fn, exc):
         compare_numba_and_py(out_fg, input_vals)
 
 
+@pytest.mark.xfail(reason="Error message not triggered")
```

Comment:
@aseyboldt the numba Elemwise code doesn't seem to trigger this branch: https://github.com/pymc-devs/pytensor/blob/5c87d7419c7208288337a5c87dad3e4c8ad3bdb0/pytensor/link/numba/dispatch/elemwise_codegen.py#L42-L48

Any idea why?

Reply:
Ah, I think it missed the case where the invalid 1 was the reference axis and not the one being compared to.